packages/orbit-components/src/Modal/README.md

```diff
@@ -25,6 +25,7 @@ Table below contains all types of the props available in the Modal component.
 | Name                | Type                       | Default    | Description                                                                                                                                                                                           |
```

Comment:
This file contains Accessibility section, I guess it can be removed now?

Reply:
I guess you're referring to the Accessibility section. I'll remove it.